src/main/java/frc/robot/subsystems/SwerveModule.java

```diff
@@ -68,6 +68,12 @@ public class SwerveModule extends SubsystemBase {
 
     private final String ID;
 
+    private DataLog dataLog = DataLogManager.getLog();
+    private DoubleLogEntry speedLog;
+    private DoubleLogEntry rotationLog;
+    private DoubleLogEntry setSpeedLog;
+    private DoubleLogEntry setRotationLog;
```

Comment:
Only rotationLog is used. Add logging for the other ones?

Reply:
speedLog, setSpeedLog, and setRotationLog were already being logged in other variables so we removed that, and rotationLog was actually logging the speed of rotation so we renamed the variable,